vocab/credentials.jsonld

```diff
@@ -0,0 +1,251 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "cred": "https://w3id.org/credentials#",
+    "dc": "http://purl.org/dc/terms/",
+    "schema": "http://schema.org/",
+    "sec": "https://w3id.org/security#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "CryptographicKey": "sec:Key",
+    "GraphSignature2012": "sec:GraphSignature2012",
+    "Identity": "https://w3id.org/identity#Identity",
+    "LinkedDataSignature2015": "sec:LinkedDataSignature2015",
+    "created": {
+      "@id": "dc:created",
+      "@type": "xsd:dateTime"
+    },
+    "creator": {
+      "@id": "dc:creator",
+      "@type": "@id"
+    },
+    "domain": "sec:domain",
+    "expires": {
+      "@id": "sec:expiration",
+      "@type": "xsd:dateTime"
+    },
+    "nonce": "sec:nonce",
+    "normalizationAlgorithm": "sec:normalizationAlgorithm",
+    "owner": {
+      "@id": "sec:owner",
+      "@type": "@id"
+    },
+    "privateKey": {
+      "@id": "sec:privateKey",
+      "@type": "@id"
+    },
+    "privateKeyPem": "sec:privateKeyPem",
+    "publicKey": {
+      "@id": "sec:publicKey",
+      "@type": "@id"
+    },
+    "publicKeyPem": "sec:publicKeyPem",
+    "publicKeyService": {
+      "@id": "sec:publicKeyService",
+      "@type": "@id"
+    },
+    "revoked": {
+      "@id": "sec:revoked",
+      "@type": "xsd:dateTime"
+    },
+    "signature": "sec:signature",
+    "signatureAlgorithm": "sec:signatureAlgorithm",
+    "signatureValue": "sec:signatureValue",
+    "Credential": "cred:Credential",
+    "claim": {
+      "@id": "cred:claim",
+      "@type": "@id"
+    },
+    "credential": {
+      "@id": "cred:credential",
+      "@type": "@id"
+    },
+    "issued": {
+      "@id": "cred:issued",
+      "@type": "xsd:dateTime"
+    },
+    "issuer": {
+      "@id": "cred:issuer",
+      "@type": "@id"
+    },
+    "recipient": {
+      "@id": "cred:recipient",
+      "@type": "@id"
+    },
+    "referenceId": {
+      "@id": "cred:referenceId"
+    }
+  },
+  "@graph": {
```

Comment:
I'm wondering if we should split this out, where doing https://w3id.org/credentials gets this document, and https://w3id.org/credentials/v1 just gets the context file w/o the RDFs/OWL vocabulary description. I'm concerned about file size, but perhaps unnecessarily so. I know that we always cache local copies in our programs and almost never go out to the network to get known contexts.

Reply:
I'll update the script to create both the all-inclusive vocabulary and a separate context-only file.